### PR TITLE
adding initial framework of store, actions, reducers and app

### PR DESCRIPTION
**Pull request by erindepew**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Copyright © 2016 Catalin Luntraru
```
Comment (mccorkle) [inline LICENSE:1]: From https://redux-minimal.js.org/ ???  Can you throw something about that into the README -- she has good "get up and running" on her front page :-)

**Comment by mccorkle** [comment]
Lookin good to me.

screen shot 2017-01-26 at 10 40 33 pm

Put a quick note in the readme for redux-minimal and I'm happy to +1 this.  

Don't know if you've got privs to merge into SEWebFE or not, but if not, I'll merge after that.

File: LICENSE (hunk 1 above)
Comment (erindepew) [reply LICENSE:1]: Yeah, I really like her templates when I don't want to hassle with setting up build tasks and what not :) I'll link to her site and the original repository.

**Comment by erindepew** [comment]
merging...

**Pull request merged**